bluebrain/repo-patches/packages/hpctoolkit/package.py

```diff
@@ -0,0 +1,126 @@
+from spack.package import *
```

Comment:
@matz-e / @heerener We did not want to update the original recipe with the latest changes in the official repo, so we added a patch to prevent messing up with your version.

Reply:
(This is referring to the whole file, not that particular line :sweat_smile:)

Reply:
@sergiorg-hpc : so, this new file is copied from upstream's develop branch without any of our custom changes?

Reply:
Yup, that is correct, I was just explaining that to @heerener.

I remember @matz-e telling me back in the day not to meddle with the original Spack recipes, so that he can update everything at once when he fetches from upstream (this is really back in the day, 1.5 years ago maybe, so maybe this is no longer valid)

Since then, I have always added patches with new versions. For instance, I did that with `arm-forge` and other recipes.

If you are fine with the idea of updating the full package specification, @kotsaloscv and I have no objection. We were playing safe here based on what I remembered from @matz-e.

Reply:
Ok, understood. I don't know our strategy or recommendation in 2023. Let's wait for a blessing from @matz-e then!

Reply:
After clarifying discussion, the idea is: "don't modify anything in `builtin` that you are not prepared to lose." 
Backporting `upstream/develop` recipes is absolutely fine and even preferred (keeps number of patches lower, and won't interfere / get lost with the next upgrade).

Reply:
what @heerener said.